packages/fxa-payments-server/src/lib/formats.ts

```diff
@@ -171,6 +171,32 @@ export function formatPlanPricing(
   }
 }
 
+/**
+ * Helper function to format a Stripe plan's interval only
+ * Examples:
+ *   'daily' or 'days'
+ *   'weekly' or 'weeks'
+ *   'monthly' or 'months'
+ *   'yearly' or 'years'
+ * @param interval
+ * @param intervalCount
+ */
+export function formatPlanInterval(
```

Comment:
I noticed this is missing some tests.

Also, since 1 is being hard coded, does this method need `intervalCount`? Removing it would simplify the function and tests a bit.

Reply:
Haven't done tests yet. I was waiting for the blocking ticket first before doing them altogether but good reminder, so thank you!

I think having a generic function for just the frequencies would be good. `formatPlanPricing` is more restrictive, as it includes pricing. But let me know your thoughts. I know a lot of things are changing for 3.0 so if it's not worth the extra effort INHO, then I can update as suggested.